### PR TITLE
Resolve deploy issues for linkedin verification

### PR DESCRIPTION
**Pull request by techadam**

### CODE REVIEW

**Review by careerbuddytech — APPROVED** [review]

File: src/views/Jobs.vue
Comment (careerbuddytech) [inline src/views/Jobs.vue:1]: Refer to the usage readme for possible search fields, if not clear in anyone, ask backend for explanation.

File: src/views/auth/Onboard.vue
Comment (careerbuddytech) [inline src/views/auth/Onboard.vue:1]: This is what I was talking about, this is fine now.

File: src/utils/helper.js
Comment (careerbuddytech) [inline src/utils/helper.js:1]: Add samesite parameter to the cookie for it to be valid across host domains, so it won’t be treated as lax.

**Review by careerbuddytech — DISMISSED** [review]
The helpers.js and Jobs.vue need attention as commented on each file.

File: src/views/Jobs.vue
Comment (techadam) [reply src/views/Jobs.vue:1]: @onyijne which one is usage readme. I don't quite get it

Comment (onyijne) [reply src/views/Jobs.vue:1]: https://github.com/CareerBuddyTalent/careerbuddy-backend/blob/main/documentation/usage.md

**Review by onyijne — APPROVED** [review]

**Pull request merged**